### ISSUE TITLE
Implement custom internal calls

### ISSUE THREAD

**Issue by LazyDuchess** [LazyDuchess/MonoPatcher#2]
Allow people to register and run their own C++ methods via DllImport. `mono_add_internal_call` allows this.

**Comment by just-harry** [comment]
Is there anything blocking this from being released?
If there is, I'd be happy to lend a hand—if I can, otherwise I'll wait patiently :)

**Comment by LazyDuchess** [comment]
> Is there anything blocking this from being released? If there is, I'd be happy to lend a hand—if I can, otherwise I'll wait patiently :)

Not at all! I had some plans but I think they're better left for another version (Prefixes, Postfixes, nicer IL editing)

I'll be releasing an update with the new plugin stuff soon, thanks for the contributions!

**Comment by just-harry** [comment]
That was soon indeed, thank you!
